bitcoin/src/blockdata/script.rs

```diff
@@ -987,9 +987,16 @@ impl Builder {
         self
     }
 
-    /// Adds an `OP_VERIFY` to the script, unless the most-recently-added
-    /// opcode has an alternate `VERIFY` form, in which case that opcode
-    /// is replaced e.g., `OP_CHECKSIG` will become `OP_CHECKSIGVERIFY`.
+    /// Adds an `OP_VERIFY` to the script or replaces the last opcode with VERIFY form
```

Comment:
For my learning; is there a reason you do not put a full stop on the brief description?

Reply:
I guess a bad habit from git.

Reply:
Ahh, could be :)